docs/how_to/users_api.rst

```diff
@@ -0,0 +1,345 @@
+Users API Documentation
+=======================
+
+**Get User**
+
+URL: ``/eox-core/api/v1/user/``
+
+Method: GET
+
+Query Params:
+
+    username (String)
+
+    email (String)
+
+Example:
+
+   http://tenant-a.local.edly.io:8000/eox-core/api/v1/user/?username=johndoe
```

Comment:
Could we use a http code-block like in the update user section?

Reply:
Done